frontend/src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import styles from './notification.module.css';
+import { notificationBadge, notificationBell } from '../../icons/other';
+
+const Card = () => (
+    <div className={styles.container}>
+      <img src={notificationBell} className={styles.bell} alt="notification bell icon" />
+      <img src={notificationBadge} className={styles.badge} alt="notification badge icon" />
```

Comment:
can you add state to make this show up only when notifications are present? the notification logic doesnt need to be implemented, just the state

Reply:
yeap! will do